batch/batch/log_store.py

```diff
@@ -11,50 +11,76 @@
 
 
 class LogStore:
-    def __init__(self, batch_logs_bucket_name, instance_id, blocking_pool, *, project=None, credentials=None):
+    def __init__(self, batch_logs_bucket_name, instance_id, *, credentials=None):
```

Comment:
I prefer that we don't deeply integrate a particular choice of FS with the LogStore. Let's have the LogStore take an `fs: FS` argument instead.